### DISCUSSION TITLE
Help ：In basic orchestration mode, the conversation opener（Opening remarks） function does not work

### DISCUSSION THREAD

**Discussion by Smile-L** [langgenius/dify#9781] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

In basic orchestration mode, the conversation opener function does not work

**Comment by Smile-L** [comment]
sorry，i found out how  to use this function.

**Reply by lvenle** [reply]
hello，can you tell me how to resole the problem?
